meson.build

```diff
@@ -62,6 +62,9 @@ endif
 
 if get_option('use_system_fltk')
     libfltk_dep = dependency('fltk', version: '>=1.4', required: false)
+    if not libfltk_dep.found()
+      libfltk_dep = cc.find_library('fltk', required: false)
```

Comment:
Finds it on Manjaro but not in the macOS runner.

@KaruroChori See https://github.com/vbeffara/Simulations/blob/8af861e80d8520e10c9daebd2d548615a45c170c/libvb/meson.build#L14 you can use `fltk-config`

Reply:
I have been explicitly told not to upstream, at least not with >=1.4

Reply:
> For ref [fltk/fltk#1148](https://github.com/fltk/fltk/issues/1148) Searching it via cmake (which meson should do) is supposed to work.

I opened a discussion about it: https://github.com/fltk/fltk/discussions/1169